source/deployment/ldap-group-sync.rst

```diff
@@ -72,7 +79,7 @@ Adding Default Teams or Channels For The Group
 --------------------------------------------
 To add the teams and channels that you want the group members to default in, select either **Add Team** or **Add Channel** from the **Add Team or Channel** button.
 
-.. image:: ../images/Add_Team_Or_Channel.png
+.. image:: ../images/Group_Configuration.png
```

Comment:
Was the image added to source/images file?

Reply:
I though

> Where is the Add_Team_Channel_Role.png image used?

It's been removed - should I remove it from the images directory or do we retain for reference purposes?

Reply:
I thought it was but I can't find it so I must have uploaded it somewhere else. Will rectify.